### PR TITLE
Removing training and validation loss and adding training time as the end.

### LINKED ISSUE #2673: Activity classifier adds training/validation loss to the display
For an activity classifier, here is the output from master
```
Pre-processing 542784 samples...
Using sequences of size 1000 for model creation.
Processed a total of 44 sessions.
Using GPU to create model (AMD Radeon Pro 455)
+--------------+----------------+--------------+---------------------+-----------------+--------------+
| Iteration    | Train Accuracy | Train Loss   | Validation Accuracy | Validation Loss | Elapsed Time |
+--------------+----------------+--------------+---------------------+-----------------+--------------+
| 1            | 0.598022       | 1.02007      | 0.794667            | 1.171           | 3.76s        |
| 2            | 0.792019       | 0.580599     | 0.866161            | 0.575019        | 4.56s        |
| 3            | 0.840314       | 0.430151     | 0.886644            | 0.328279        | 5.93s        |
| 4            | 0.864244       | 0.352443     | 0.894552            | 0.269271        | 7.44s        |
| 5            | 0.877734       | 0.317695     | 0.904325            | 0.245527        | 8.96s        |
| 6            | 0.8847         | 0.294302     | 0.894541            | 0.252827        | 10.49s       |
| 10           | 0.917601       | 0.199895     | 0.922801            | 0.184124        | 16.74s       |
+--------------+----------------+--------------+---------------------+-----------------+--------------+
0.8824701370055381
```

Output from 5.8

```
Pre-processing 634469 samples...
Using sequences of size 1000 for model creation.
Processed a total of 51 sessions.
Using GPU to create model (AMD Radeon Pro 455)
+----------------+----------------+----------------+----------------+
| Iteration      | Train Accuracy | Train Loss     | Elapsed Time   |
+----------------+----------------+----------------+----------------+
| 1              | 0.644          | 0.984          | 1.5            |
| 2              | 0.823          | 0.526          | 2.5            |
| 3              | 0.861          | 0.376          | 3.4            |
| 4              | 0.880          | 0.322          | 4.2            |
| 5              | 0.898          | 0.276          | 5.1            |
| 6              | 0.903          | 0.252          | 6.0            |
| 7              | 0.907          | 0.243          | 6.9            |
| 8              | 0.922          | 0.210          | 8.2            |
| 9              | 0.917          | 0.196          | 9.5            |
| 10             | 0.927          | 0.190          | 10.8           |
+----------------+----------------+----------------+----------------+
Training complete
Total Time Spent: 10.7528s
0.877783336405
```

### PR DESCRIPTION
**Pull request by shreyajain17**
Display matches 5.8 - displaying the losses

```
In [2]: import turicreate as tc                                                                                                   

In [3]: data = tc.SFrame('/Users/shreya_jain3/Downloads/hapt_data.sframe/')                                                       

In [4]: model = tc.activity_classifier.create(data, 'exp_id', 'activity')                                                         
Pre-processing 658987 samples...
Using sequences of size 2000 for model creation.
Processed a total of 54 sessions.
Using GPU to create model (AMD Radeon Pro 560X)
+--------------+----------------+---------------------+--------------+
| Iteration    | Train Accuracy | Validation Accuracy | Elapsed Time |
+--------------+----------------+---------------------+--------------+
| 1            | 0.548125       | 0.792035            | 12.79s       |
| 2            | 0.780194       | 0.817142            | 14.50s       |
| 3            | 0.845181       | 0.823762            | 16.19s       |
| 4            | 0.872553       | 0.849104            | 17.93s       |
| 5            | 0.891892       | 0.864335            | 19.59s       |
| 6            | 0.906368       | 0.868652            | 21.45s       |
| 10           | 0.921173       | 0.869703            | 29.17s       |
+--------------+----------------+---------------------+--------------+
Training complete
Total Time Spent: 53.6294
```

fixes #2673

### CODE REVIEW

File: src/python/turicreate/toolkits/activity_classifier/_activity_classifier.py [hunk 1]
```diff
@@ -186,6 +186,7 @@ def create(dataset, session_id, target, features=None, prediction_window=100,
         options['batch_size'] = batch_size
         options['max_iterations'] = max_iterations
         options['verbose'] = verbose
+        options['show_loss'] = False
```
Comment (nickjong) [inline src/python/turicreate/toolkits/activity_classifier/_activity_classifier.py]: Could we use a leading underscore here, to make it more clear that this is (for now) an internal option?

File: src/toolkits/activity_classification/activity_classifier.cpp [hunk 2]
```diff
@@ -398,6 +418,8 @@ void activity_classifier::train(
   state_update["training_time"] = time_object.current_time();
 
   add_or_update_state(state_update);
+  std::cout << "Training complete" << std::endl;
```
Comment (nickjong) [inline src/toolkits/activity_classification/activity_classifier.cpp]: We should be using `logprogress_stream` instead of `std::cout` for this kind of thing, I believe. See examples in OD and in supervised learning, e.g. logistic regression
